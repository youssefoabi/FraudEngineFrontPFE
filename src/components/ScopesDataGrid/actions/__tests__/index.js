import { itShouldTestActionCreators } from '@cdiscount/jest-utils';

import * as actions from '../';

describe('Scopes actions', () => {
  itShouldTestActionCreators(actions);
});
