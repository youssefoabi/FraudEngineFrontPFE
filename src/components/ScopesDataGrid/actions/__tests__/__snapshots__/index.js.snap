// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Scopes actions DELETE_SCOPES_ERROR 1`] = `"@SCOPES/DELETE_SCOPES_FAILURE"`;

exports[`Scopes actions DELETE_SCOPES_REQUEST 1`] = `"@SCOPES/DELETE_SCOPES_REQUEST"`;

exports[`Scopes actions DELETE_SCOPES_SUCCESS 1`] = `"@SCOPES/DELETE_SCOPES_SUCCESS"`;

exports[`Scopes actions FETCH_SCOPES 1`] = `"@SCOPES/FETCH"`;

exports[`Scopes actions RESET_DELETE_SCOPES_ERROR 1`] = `"@SCOPES/RESET_DELETE_SCOPES_ERROR"`;

exports[`Scopes actions RESET_DELETE_SCOPES_SUCCESS 1`] = `"@SCOPES/RESET_DELETE_SCOPES_SUCCESS"`;

exports[`Scopes actions SET_SCOPES 1`] = `"@SCOPES/SET"`;

exports[`Scopes actions SET_SCOPES_PAGE 1`] = `"@SCOPES/SET_PAGE"`;

exports[`Scopes actions SET_SCOPES_ROWS_PER_PAGE 1`] = `"@SCOPES/SET_ROWS_PER_PAGE"`;

exports[`Scopes actions SET_SCOPES_SELECTED 1`] = `"@SCOPES/SET_SELECTED"`;

exports[`Scopes actions SET_SCOPES_SORT 1`] = `"@SCOPES/SET_SORT"`;
